Source/Applications/SystemCenterNotification/Scripts/TSX/Report/ByReport.tsx

```diff
@@ -126,9 +127,11 @@ const ByReport = (props: IProps) => {
                             <fieldset className="border" style={{ padding: '10px', height: '100%' }}>
                                 <legend className="w-auto" style={{ fontSize: 'large' }}>Actions:</legend>
                                 <form>
-                                    <button className="btn btn-primary" onClick={(event) => { setNewEmail(emptyEmail); setShowModal(true); event.preventDefault() }}>
```

Comment:
do we really need a `form`? Seems like a `div` would be better suited with the appropriate class to remove the `<br />`